deduction_rules.py

```diff
@@ -39,15 +39,15 @@ def _apply_for_nines(func):
     made_deduction = False
     for row in range(9):
         cells_to_check = [(row,col) for col in range(9)]
-        made_deduction |= func(cells_to_check)
+        made_deduction |= func(cells_to_check, {"type": "row", "idx": row})
 
     for col in range(9):
         cells_to_check = [(row,col) for row in range(9)]
-        made_deduction |= func(cells_to_check)
+        made_deduction |= func(cells_to_check, {"type": "col", "idx": row})
```

Comment:
`"idx": col` should be here.
![image](https://user-images.githubusercontent.com/69110182/144314477-bbdede87-d5d3-4c1e-92f2-7c032b0b5b21.png)

Reply:
It is already fixed I think.

Reply:
But the print screen look sus.